src/util.cpp

```diff
@@ -830,17 +830,27 @@ static std::vector<std::pair<std::string, std::string>> GetConfigOptions(std::is
                 std::string name = prefix + TrimString(str.substr(0, pos), pattern);
                 std::string value = TrimString(str.substr(pos + 1), pattern);
                 options.emplace_back(name, value);
+            } else {
+                error = strprintf("parse error on line %i: %s", linenr, str);
+                if (str.size() >= 2 && str.substr(0, 2) == "no") {
+                    error += strprintf(", if you intended to specify a negated option, use %s=1 instead", str);
```

Comment:
we could also change this logic here to interpret it as `noX=1`, but I'm not sure what kind of syntax we want to support

Reply:
error = strprintf ( "แยกวิเคราะห์ข้อผิดพลาดในบรรทัด% i:% s " , linenr, str);
                if (str. size ()> = 2 && str. substr ( 0 , 2 ) == " noX=1 " ) {
                    ข้อผิดพลาด + = strprintf ( "ถ้าคุณตั้งใจจะระบุตัวเลือกที่ถูกละเลยให้ใช้% s = 1 แทน" , str);